user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectSummaryComponent.java

```diff
@@ -447,6 +457,25 @@ private Div renderContactInfo(Contact contact) {
     var name = new Span(contact.fullName());
     var email = new Anchor("mailto:" + contact.emailAddress(), contact.emailAddress());
     contactInfo.add(name, email);
+    //Account for contacts without oidc or oidcissuer set
+    if (contact.oidc() == null || contact.oidcIssuer() == null) {
+      return contactInfo;
+    }
+    if (contact.oidcIssuer().isEmpty() || contact.oidc().isEmpty()) {
+      return contactInfo;
+    }
+    var oidcType = Arrays.stream(OidcType.values())
+        .filter(ot -> ot.getIssuer().equals(contact.oidcIssuer()))
+        .findFirst();
+    if (oidcType.isPresent()) {
+      String oidcUrl = String.format(oidcType.get().getUrl()) + contact.oidc();
```

Comment:
As the format of the urls might differ based on the OIDC type, I suggest moving this to the enum instead. oidcType.getUrlFor(contact.oidc())

Reply:
That's a good point i adjusted it 👍